### DISCUSSION TITLE
Best way to deal with Serverless and Turborepo

### DISCUSSION THREAD

**Discussion by zacharykapank** [vercel/turborepo#558] [Help] [answered]
I just recently switched to turbo repo within my projects and ran into some issues with apps making use of serverless. The issue lies with serverless not packaging modules that are hoisted as shared dependencies, which caused some breaking changes. 
I have been trying to find a neat solution for this and still have the benefit of speed with turbo repo. Tried some serverless packages for monorepo but it didn't solve the issue I was having.

**Comment by anthonyshew** [comment, marked as answer]
This is a runtime infrastructural limitation and not in the scope of `turbo`.
